pkgs/by-name/qw/qwerty-fr/package.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, fetchurl
+, lib
+, unzip
+,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qwerty-fr";
+  version = "0.7.3";
+
+  src = fetchurl {
+    url = "https://github.com/qwerty-fr/qwerty-fr/releases/download/v${version}/qwerty-fr_${version}_linux.zip";
+    sha256 = "0csxzs2gk8l4y5ii1pgad8zxr9m9mfrl9nblywymg01qw74gpvnm";
+  };
```

Comment:
This should use [`fetchFromGitHub`](https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub) instead

Reply:
```suggestion
  src = fetchFromGitHub {
    owner = "qwerty-fr";
    repo = "qwerty-fr";
    rev = "refs/tags/v${finalAttrs.version}";
    sha256 = "sha256-BZLp5Tw1BH/m1wHYAUbfP86uZ6poAjD2D/uvl2ajmi0=";
  };
```